README.md

```
@@ -147,18 +147,38 @@ Experiment:
+  make build
+  ```
+  ### Запуск всей системы
+  ```
+  make up
+  ```
+  ### Запуск frontend-части ```make up SERVICE=frontend```
+  ### Запуск etl-инструмента ```make up SERVICE=etl-service```
+  ### Запуск testing-system ```make up SERVICE=testing-system```
+
+  ### Отключение ```make down```
+  ### Просмотр логов в винтерактивном формате ```make logs SYSTEM=<system to show logs from, optional>```
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)